code/parse/sexp.cpp

```diff
@@ -3865,8 +3865,9 @@ void stuff_sexp_text_string(SCP_string &dest, int node, int mode)
 				Assert(mode == SEXP_SAVE_MODE);
 				sprintf(dest, "@%s[%s] ", Sexp_nodes[node].text, Sexp_variables[sexp_variables_index].text);
 			}
-		} else {
-			// string
+		}
+		// string
+		else {
```

Comment:
I don't like commenting on formatting but this style (else on the next line) does not fit with the code above so that should be made consistent.

Reply:
I wanted to make the minimal format change to clarify the comment, but...